pennylane/__init__.py

```diff
@@ -166,14 +166,9 @@ class PennyLaneDeprecationWarning(UserWarning):
     """Warning raised when a PennyLane feature is being deprecated."""
 
 
-del globals()["Hamiltonian"]
-
-
 def __getattr__(name):
     if name == "Hamiltonian":
-        if pennylane.operation.active_new_opmath():
-            return pennylane.ops.LinearCombination
-        return pennylane.ops.Hamiltonian
+        return pennylane.ops.LinearCombination
```

Comment:
Could we just do `from pennylane.ops import LinearCombination as Hamiltonian` outside of the `getattr`?